prefork_test.go

```diff
@@ -23,15 +23,15 @@ func Test_App_Prefork_Child_Process(t *testing.T) {
 
 	app := New()
 
-	err := app.prefork("invalid", nil)
+	err := app.prefork("tcp4", "invalid", nil)
 	utils.AssertEqual(t, false, err == nil)
 
 	go func() {
 		time.Sleep(1000 * time.Millisecond)
 		utils.AssertEqual(t, nil, app.Shutdown())
 	}()
 
-	utils.AssertEqual(t, nil, app.prefork("[::]:", nil))
+	utils.AssertEqual(t, nil, app.prefork("tcp6", "[::]:", nil))
```

Comment:
Wouldn't it be better if you keep this strings (`tcp`, `tcp4`, `tcp6`) as constant and use the constant everywhere?
Also make those constants exported, so that when I use the network option, i can use it like `Network: fiber.NetworkTCP4` instead of `Network: "tcp4"`. And typing hardcoded strings can be misspelled.